src/main/java/com/ubb/zenith/model/UserPlaylist.java

```diff
@@ -0,0 +1,63 @@
+package com.ubb.zenith.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Objects;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserPlaylist {
+    @Embeddable
+    public static class UserPlaylistsId implements Serializable {
+        private Integer userId;
+        private Integer playlistId;
+
+        // Default constructor
+        public UserPlaylistsId() {}
+
+        // Parameterized constructor
+        public UserPlaylistsId(Integer userId, Integer playlistId) {
```

Comment:
Same for this constructor, you already use @AllArgsConstructor. You don't have to write parametrizied constructor